runtime/rasdump/rasdump_internal.h

```diff
@@ -112,10 +112,12 @@ void setAllocationThreshold(J9VMThread *vmThread, UDATA min, UDATA max);
 #define J9RAS_SYSTEMINFO_HYPERVISOR         2
 #define J9RAS_SYSTEMINFO_CORE_PATTERN       3
 #define J9RAS_SYSTEMINFO_CORE_USES_PID      4
+#define J9RAS_SYSTEMINFO_CORE_PATTERN2      5
 
 #define J9RAS_SCHED_COMPAT_YIELD_FILE "/proc/sys/kernel/sched_compat_yield"
 #define J9RAS_CORE_PATTERN_FILE "/proc/sys/kernel/core_pattern"
 #define J9RAS_CORE_USES_PID_FILE "/proc/sys/kernel/core_uses_pid"
+#define J9RAS_CORE_PATTERN_FILE2 ".../oneagent/agent/conf/original_core_pattern"
```

Comment:
This isn't the best name - it's not a file name. Suggest `J9RAS_CORE_ORIGINAL_PATTERN`.